tests/unit/test_stages/test_base_stage.py

```diff
@@ -151,6 +150,28 @@ def test_dashboard_view(self, workgroups, target_students, state, stats):
         self.assertDictionaryEqual(args[1], expected_context)
 
     @ddt.data(
+        (make_stats(0.4, 0.3, 0.4), (40, 30, 40)),
+        (make_stats(0.1, 0.1, 0.8), (80, 10, 10)),
+        (make_stats(None, None, None), (None, None, None))
+    )
+    @ddt.unpack
+    def test_make_human_stats(self, stats, human_stats_data):
+        stats_order = (StageState.NOT_STARTED, StageState.INCOMPLETE, StageState.COMPLETED)
+        actual_human_stats = BaseGroupActivityStage.make_human_stats(stats)
+
+        expected_human_stats = OrderedDict([
+            (StageState.get_human_name(stats_order[idx]), human_stats_data[idx])
+            for idx in range(3)
+        ])
+
+        self.assertEqual(len(actual_human_stats), 3)
```

Comment:
@jbzdak nit: since both `actual_` and `expected_` are dicts, you can employ stricter check:

    self.assertEqual(actual_human_stats.keys(), expected_human_stats.keys())